packages/cyberstorm/src/components/Layout/Developers/PackageUpload/PackageUploadLayout.tsx

```diff
@@ -34,14 +36,19 @@ export function PackageUploadLayout() {
             title="Team"
             description="No teams available?"
             additionalLeftColumnContent={
-              <div>
-                <Button.Root>
-                  <Button.ButtonLabel>Create Team</Button.ButtonLabel>
-                  <Button.ButtonIcon>
-                    <FontAwesomeIcon icon={faPlus} />
-                  </Button.ButtonIcon>
-                </Button.Root>
-              </div>
+              <Dialog.Root
+                title="Create Team"
+                trigger={
+                  <Button.Root colorScheme="primary" paddingSize="large">
```

Comment:
I'm not sure this matches the designs in Figma?

Reply:
Did not before, does now 👍